README.md

```diff
@@ -728,7 +728,6 @@ Follows best practices and conventions to provide you a SOLID development experi
 * [EntryPoint](https://github.com/Nick-Lucas/EntryPoint) - Composable CLI (Command Line) Argument Parser for .Net Core & .Net Framework 4.5+.
 * [Fake JSON Server](https://github.com/ttu/dotnet-fake-json-server) - Fake REST API for prototyping or as a CRUD Back End. No need to define types, uses dynamic typing. Data is stored to a single JSON file. Has authentication, WebSocket notifications, async long running operations, random generation for errors/delays and experimental GraphQL support.
 * [gitignore.io](https://github.com/joeblau/gitignore.io) - Create useful .gitignore files for your project [https://www.gitignore.io](https://www.gitignore.io).
-* [ICanHasDotnetCore](https://github.com/OctopusDeploy/ICanHasDotnetCore) - Scans uploaded packages.config files or GitHub repository and determines whether the nuget packages target .NET Standard [https://icanhasdot.net](https://icanhasdot.net).
```

Comment:
Why do you remove it ? The link still work

Reply:
Cause Travis claimed that the link did not work. I tried it manually too two days ago and was able to confirm it.
Due to how the result of the build job will be created, all new PRs are failing.
So I tried to fix this issue directly in my PR.

Reply:
I readded it instead of a revert. Hope that`s ok..
Let`s see if Travis builds now.